### PR TITLE
test coverage checking

### PR DESCRIPTION
**Pull request by benadida**
We should check python test coverage and enforce it. This is a PR to get there.

### CODE REVIEW

**Comment by eventualbuddha** [comment]
This doesn't actually enforce anything yet, right?

File: Makefile [hunk 1]
```diff
@@ -42,3 +42,6 @@ test-client:
 # To run a specific test: TEST=<test name> make test-server
 test-server:
 	FLASK_ENV=test python3 -m pipenv run python -m pytest -k '${TEST}' --ignore=arlo-client
+
+test-server-coverage:
+	python3 -m pipenv run python -m pytest --cov=arlo_server/
```
Comment (eventualbuddha) [inline Makefile]: Do we need `FLASK_ENV=test`? what about `--ignore=arlo-client`? Do we want `-k '${TEST}'`?

Comment (benadida) [reply Makefile]: I don't think we should do coverage with paring down to one test, but yeah I think we need to tweak some other settings before we merge. I should have marked this as draft. Not planning on merging until we at least figure out how to cover everything properly.

**Comment by jonahkagan** [comment]
Closing in favor of #451

**Pull request closed without merging**